2014-02-05 07:45:07,905 [main] ERROR  partnerIntg.App - UUID for the Job is : 2397574782064087001
2014-02-05 07:50:07,381 [main] ERROR  partnerIntg.App - UUID for the Job is : 8381248903225428081
2014-02-05 14:19:36,600 [main] ERROR  partnerIntg.App - UUID for the Job is : -6717753439392545113
2014-02-05 14:19:37,236 [main] ERROR  partnerIntg.App - Error in XML validation
org.xml.sax.SAXParseException: cvc-complex-type.2.4.a: Invalid content was found starting with element 'paymentTerms'. One of '{sourceSystem}' is expected.
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:198) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.error(ErrorHandlerWrapper.java:134) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:437) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:368) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:325) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.impl.xs.XMLSchemaValidator$XSIErrorReporter.reportError(XMLSchemaValidator.java:458) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.impl.xs.XMLSchemaValidator.reportSchemaError(XMLSchemaValidator.java:3237) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.impl.xs.XMLSchemaValidator.handleStartElement(XMLSchemaValidator.java:1796) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.impl.xs.XMLSchemaValidator.startElement(XMLSchemaValidator.java:746) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.scanStartElement(XMLNSDocumentScannerImpl.java:376) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:2768) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:606) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:116) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:511) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:846) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:775) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.jaxp.validation.StreamValidatorHelper.validate(StreamValidatorHelper.java:151) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.jaxp.validation.ValidatorImpl.validate(ValidatorImpl.java:116) ~[na:1.7.0_40]
	at javax.xml.validation.Validator.validate(Validator.java:124) ~[na:1.7.0_40]
	at partnerIntg.App.validateAgainstXSD(App.java:52) [classes/:na]
	at partnerIntg.App.main(App.java:94) [classes/:na]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.7.0_40]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57) ~[na:1.7.0_40]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.7.0_40]
	at java.lang.reflect.Method.invoke(Method.java:606) ~[na:1.7.0_40]
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:120) [idea_rt.jar:na]
2014-02-05 14:19:37,321 [main] ERROR  partnerIntg.App - Error in XML validation
org.xml.sax.SAXParseException: cvc-complex-type.2.4.a: Invalid content was found starting with element 'paymentTerms'. One of '{sourceSystem}' is expected.
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:198) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.error(ErrorHandlerWrapper.java:134) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:437) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:368) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.impl.XMLErrorReporter.reportError(XMLErrorReporter.java:325) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.impl.xs.XMLSchemaValidator$XSIErrorReporter.reportError(XMLSchemaValidator.java:458) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.impl.xs.XMLSchemaValidator.reportSchemaError(XMLSchemaValidator.java:3237) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.impl.xs.XMLSchemaValidator.handleStartElement(XMLSchemaValidator.java:1796) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.impl.xs.XMLSchemaValidator.startElement(XMLSchemaValidator.java:746) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.scanStartElement(XMLNSDocumentScannerImpl.java:376) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next(XMLDocumentFragmentScannerImpl.java:2768) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next(XMLDocumentScannerImpl.java:606) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.impl.XMLNSDocumentScannerImpl.next(XMLNSDocumentScannerImpl.java:116) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument(XMLDocumentFragmentScannerImpl.java:511) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:846) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse(XML11Configuration.java:775) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.jaxp.validation.StreamValidatorHelper.validate(StreamValidatorHelper.java:151) ~[na:1.7.0_40]
	at com.sun.org.apache.xerces.internal.jaxp.validation.ValidatorImpl.validate(ValidatorImpl.java:116) ~[na:1.7.0_40]
	at javax.xml.validation.Validator.validate(Validator.java:124) ~[na:1.7.0_40]
	at partnerIntg.App.validateAgainstXSD(App.java:52) [classes/:na]
	at partnerIntg.App.main(App.java:94) [classes/:na]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.7.0_40]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57) ~[na:1.7.0_40]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.7.0_40]
	at java.lang.reflect.Method.invoke(Method.java:606) ~[na:1.7.0_40]
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:120) [idea_rt.jar:na]
2014-02-05 14:21:46,954 [main] ERROR  partnerIntg.App - UUID for the Job is : -3275962219536691841
2014-02-05 14:25:33,989 [main] ERROR  partnerIntg.App - UUID for the Job is : 6044672769544636199
2014-02-05 14:28:46,385 [main] ERROR  partnerIntg.App - UUID for the Job is : -6815599664566350330
2014-02-05 14:28:46,865 [main] ERROR  partnerIntg.App - Exception in main function
org.springframework.dao.InvalidDataAccessResourceUsageException: Could not execute JDBC batch update; nested exception is org.hibernate.exception.SQLGrammarException: Could not execute JDBC batch update
	at org.springframework.orm.hibernate3.SessionFactoryUtils.convertHibernateAccessException(SessionFactoryUtils.java:630) ~[spring-2.5.6.jar:2.5.6]
	at org.springframework.orm.hibernate3.HibernateAccessor.convertHibernateAccessException(HibernateAccessor.java:412) ~[spring-2.5.6.jar:2.5.6]
	at org.springframework.orm.hibernate3.HibernateTemplate.doExecute(HibernateTemplate.java:424) ~[spring-2.5.6.jar:2.5.6]
	at org.springframework.orm.hibernate3.HibernateTemplate.executeWithNativeSession(HibernateTemplate.java:374) ~[spring-2.5.6.jar:2.5.6]
	at org.springframework.orm.hibernate3.HibernateTemplate.save(HibernateTemplate.java:694) ~[spring-2.5.6.jar:2.5.6]
	at partnerIntg.com.good.dao.PurchaseOrderDetailsDaoImpl.save(PurchaseOrderDetailsDaoImpl.java:20) ~[classes/:na]
	at partnerIntg.com.good.businessObjs.PurchaseOrderDetailsImpl.save(PurchaseOrderDetailsImpl.java:31) ~[classes/:na]
	at partnerIntg.App.main(App.java:112) ~[classes/:na]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.7.0_40]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57) ~[na:1.7.0_40]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.7.0_40]
	at java.lang.reflect.Method.invoke(Method.java:606) ~[na:1.7.0_40]
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:120) [idea_rt.jar:na]
Caused by: org.hibernate.exception.SQLGrammarException: Could not execute JDBC batch update
	at org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:67) ~[hibernate-3.2.6.ga.jar:3.2.6.ga]
	at org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:43) ~[hibernate-3.2.6.ga.jar:3.2.6.ga]
	at org.hibernate.jdbc.AbstractBatcher.executeBatch(AbstractBatcher.java:253) ~[hibernate-3.2.6.ga.jar:3.2.6.ga]
	at org.hibernate.engine.ActionQueue.executeActions(ActionQueue.java:266) ~[hibernate-3.2.6.ga.jar:3.2.6.ga]
	at org.hibernate.engine.ActionQueue.executeActions(ActionQueue.java:167) ~[hibernate-3.2.6.ga.jar:3.2.6.ga]
	at org.hibernate.event.def.AbstractFlushingEventListener.performExecutions(AbstractFlushingEventListener.java:298) ~[hibernate-3.2.6.ga.jar:3.2.6.ga]
	at org.hibernate.event.def.DefaultFlushEventListener.onFlush(DefaultFlushEventListener.java:27) ~[hibernate-3.2.6.ga.jar:3.2.6.ga]
	at org.hibernate.impl.SessionImpl.flush(SessionImpl.java:1000) ~[hibernate-3.2.6.ga.jar:3.2.6.ga]
	at org.springframework.orm.hibernate3.HibernateAccessor.flushIfNecessary(HibernateAccessor.java:390) ~[spring-2.5.6.jar:2.5.6]
	at org.springframework.orm.hibernate3.HibernateTemplate.doExecute(HibernateTemplate.java:420) ~[spring-2.5.6.jar:2.5.6]
	... 10 common frames omitted
Caused by: java.sql.BatchUpdateException: ORA-00904: "SOURCE_SYSTEM_ID": invalid identifier

	at oracle.jdbc.driver.OraclePreparedStatement.executeBatch(OraclePreparedStatement.java:10070) ~[ojdbc6-7.0.jar:11.2.0.1.0]
	at oracle.jdbc.driver.OracleStatementWrapper.executeBatch(OracleStatementWrapper.java:213) ~[ojdbc6-7.0.jar:11.2.0.1.0]
	at org.hibernate.jdbc.BatchingBatcher.doExecuteBatch(BatchingBatcher.java:48) ~[hibernate-3.2.6.ga.jar:3.2.6.ga]
	at org.hibernate.jdbc.AbstractBatcher.executeBatch(AbstractBatcher.java:246) ~[hibernate-3.2.6.ga.jar:3.2.6.ga]
	... 17 common frames omitted
2014-02-05 14:31:30,244 [main] ERROR  partnerIntg.App - UUID for the Job is : 1548082314711352469
